Sources/Protocols/MessagesLayoutDelegate.swift

```diff
@@ -237,7 +237,7 @@ public extension MessagesLayoutDelegate {
     }
 
     func shouldCacheLayoutAttributes(for message: MessageType) -> Bool {
-        return false
+        return true
```

Comment:
I just have to check this tomorrow. We had it as `false` by default because the caching of `cellTopLabel` and `cellBottomLabel` was annoying. If you had an empty label the size would get cached as `.zero`. Then if you changed the text to another value, you would get a non-zero size. But the cache was not invalidated 😅 Maybe the new equality checking will fix this

Reply:
If we can solve #504 this becomes obsolete. There is no reason to allow users to control caching in this manner. This was a hack to turn it off while the label sizes were being cached and the caching strategy was buggy. If we invalidate properly the user doesn't even need to be aware of this.